src/Spatial/Euclidean/PolyLine2D.cs

```diff
@@ -0,0 +1,224 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace MathNet.Spatial.Euclidean
+{
+    /// <summary>
+    /// The PolyLine2D class represents a 2D curve in space made up of line segments joined end-to-end, and is 
+    /// stored as a sequential list of 2D points.
+    /// </summary>
+    public class PolyLine2D : IEnumerable<Point2D>
```

Comment:
IReadonlyList<Point2D> makes sense here unless bumping framework version is an issue cc/ @cdrnet